python/rules_engine.py

```diff
@@ -2644,6 +2643,136 @@ def v708(self):
 		fail_df = self.lar_df[~(self.lar_df.business_purpose.isin(("1111", "1", "2")))]
 		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
 
+	def v709(self):
+		"""An invalid Property Address was reported. Please
+		review the information below and update your file
+		accordingly.
+		1) If Street Address, City, and Zip Code is reported
+		Exempt, then all three must be reported Exempt. """
+
+		field = "Property Address"
+		edit_name = "v709"
+		fail_df = self.lar_df[((self.lar_df.street_address=="Exempt")&(self.lar_df.city=="Exempt")&(self.lar_df.zip_code == "Exempt"))&
```

Comment:
Let's take another look at this logic. It looks like it's set to to check the full condition and the full reverse condition.

Reply:
Sure! Could this be resolved with or statements for the full condition?